int8 status
int8 goal_id